pyproject.toml

```diff
@@ -6,7 +6,11 @@ build-backend = "flit.buildapi"
 module = "gidgethub"
 author = "Brett Cannon"
 author-email = "brett@python.org"
-requires = ["uritemplate>=3.0.1"]
+requires = [
+    "uritemplate>=3.0.1",
+    "PyJWT>=1.7.1",
```

Comment:
It should've been `PyJWT[crypto]` as per their docs:
![Screenshot_2020-04-19-08-52-35-032_com android chrome](https://user-images.githubusercontent.com/578543/79681527-49140c80-821b-11ea-8bce-0fbaed3b9cd1.jpg)

Reply:
I opened https://github.com/brettcannon/gidgethub/issues/116 to track this.